style/ruby/README.md

```diff
@@ -42,6 +42,8 @@ Ruby
 * Prefer `protected` over `private` for non-public `attr_reader`s, `attr_writer`s,
   and `attr_accessor`s.
 * Order class methods above instance methods.
+* Do not include a space after the opening brace of a hash or before the closing
+  brace.
```

Comment:
>Changing the wording to something along the lines of "all uses of curly braces"

I can't see a compelling reason to have any differences depending on the location. I'll update the PR.